docs/spec/spec.md

```diff
@@ -96,7 +88,7 @@ The command will generate [Derived Entity Types and Persist Clients](https://git
 as per the `persist` specification.
 Additionally, this command will create the database schema associated with the data model definition.
 
-It will add generated files under the conventions, 
+It will add generated files under the conventions,
```

Comment:
I dont think this part is valid anymore isnt it? We generate based on Ballerina.toml persist.module value dont we?